_posts/2016-06-20-foreman-community-newsletter-june-2016.md

```diff
@@ -0,0 +1,136 @@
+with the 1.12 release (and the Puppet 4 support) I'm currently organising a
+series of events around our various geographical centres to mark the occasion.
+So far, I have events planned (but not 100% confirmed) for East-cost US, UK,
+Germany, and Israel. There'll be meetups, install days, talks and swag to be
+had! Watch [Twitter](https://twitter.com/ForemanProject) or the
+[events page](/events) to know whats confirmed!
+
+If you want to be involved in organising this effort, please contact
+[me](mailto:community@emeraldreverie.org) and I'll work something out with you
+- thanks!
+
+### Katello 3.0 (Saison) and 3.0.1 bugfix release
+
+The latest version of the Katello plugin hit it's milestone 3.0 release this
+month, after an extended RC phase - thanks again to all who contributed to the
+testing of the various RCs! The list of changes is large, so I'll just direct
+you to the [release announcement][k30] for the details. Unfortunately, one bug
+did creep though, so this was followed up with a swift [3.0.1 release][k301]
+which deals with the issue. Please do keep reporting any further issues.
+
+As the announcement states, this is a huge change to Katello, and upgrades
+require some planning - users are encouraged to read through the steps and take
+appropriate backups before upgrading. Do get in touch on IRC or the mailing
+lists if you have any questions!
+
+[k30]:  https://groups.google.com/d/topic/foreman-users/KlmzfqIcrRQ/discussion
+[k301]: https://groups.google.com/d/topic/foreman-users/vcjo8CW6p9I/discussion
+
+### SSL comes to (yum.|deb.|)theforeman.org
+
+Thanks to the hard work of the infra team, our website and packaging repos now
+have LetsEncrypt SSl certificates in place. This has the main benefit of
+allowing secure downloads of our packaging GPG keys for verification of the
+packages themselves.
+
+The quickstart guides have been [updated][gpg] to reflect the updated URL scheme.
+
+[gpg]: https://theforeman.org/manuals/1.11/quickstart_guide.html#QuickstartGuide
+
+### New media and blogs
+
+#### [Community Demo](https://youtu.be/C_BebxMpW5c) on 19th May 2016
+
+On this episode of the Foreman Community Demo, we cast a wide net to pull in
+updates from Pulp (the project that handles content for Katello), from Katello,
+from our CLI testers, and some Infoblox DNS/DHCP providers for the Smart Proxy.
+There's also a quick look at the 1.11.2 release, and a teaser for the upcoming
+Puppet 4 support!
+
+#### [Community Demo](https://youtu.be/Z2xKULg5MCA) on 9th June 2016
+
+On this episode of the Foreman Community Demo, we had a look at upcoming tools
+for flexible data exports, a new admin plugin for Hammer, some email styling
+help, new collections for Settings, and a new lock-in feature for Discovery.
+Oh, and 1.12.0 RC1 :)
+
+#### [Foreman Deep Dives: SQL Debugging for Beginners](https://www.youtu.be/kbslq-4EuyY)
+
+Chris Duryee (@beav) joined us for this Deep Dive, in which he gave us an
```

Comment:
I don't really use twitter, I would just leave it sans handle. The best way to reach me is foreman-dev channel or via the mailing list :mailbox: